SYCL/DeviceLib/ITTAnnotations/barrier.cpp

```diff
@@ -0,0 +1,45 @@
+// UNSUPPORTED: cuda
```

Comment:
why ITT is unsupported on CUDA? Is is due to AOT nature of the build?
is it supported on HIP?

Reply:
It has to do with lack of device library support. AMDGCN should actually be excluded as well - I believe this hasn't been caught by CI due to the current lack of comprehensive HIP testing.